pipeline/liminal.yml

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+name: GettingStartedPipeline
+volumes:
+  - volume: gettingstartedvol
+    claim_name: gettingstartedvol-pvc
+    local:
+      path: .
+images:
+  - image: mmrs-python
+    type: python
+    source: .
+pipelines:
+  - pipeline: getting_started_pipeline
+    owner: Bosco Albert Baracus
+    start_date: 1970-01-01
+    timeout_minutes: 10
+    schedule: 0 * 1 * *
+    default_arg_loaded: check
+    default_array_loaded: [2, 3, 4]
+    default_object_loaded:
+      key1: val1
+      key2: val2
```

Comment:
yes , these are not arguments for a task but default for all dags . This can be used intead to specify global things like on_failure_callback, on_success_callback i think like the ones of airflow https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html#default-arguments.

I think we can pass download arguments as  an http uri which contains feather file using env_vars:
``` yaml
env_vars:
     feather_file: "http://xxxx"
```

Or we can mount a read-only volume in task where feather_file will be so that we can mount it like:
``` yaml
env_vars:
     feather_file: "/mnt/d/feather_file"
```